### ISSUE TITLE
Feature Request: Add Search to Issue

### ISSUE THREAD

**Issue by HyknuF** [ThirtyDegreesRay/OpenHub#286]
Would be great to be able to search within issues Thanks!

**Comment by MCwither** [comment]
@ThirtyDegreesRay 增加搜索问题的功能

**Comment by KaKi87** [comment]
Any news on this ?

**Comment by esammier** [comment]
I agree it would be a welcome feature. Searching for an issue before creating a new one is quite common, it helps avoid duplicates

**Comment by shom13** [comment]
Any update on this? Looks like this issue has been here for years now. Its a pretty basic github functionality and should be incorporated quickly.
